### DISCUSSION TITLE
TLS certifcate

### DISCUSSION THREAD

**Discussion by BewinWilsonMathews** [community/community#68514] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

DNS check is successful but TLS certificate is not getting approved even though it has taken a long time
How to get it approved?

![tls](https://github.com/community/community/assets/95472327/79263101-438f-47c3-84e0-ab1adc22f634)

**Comment by mimiflynn** [comment]
This is also happening to me. I've even tried to remove the URL and start the process again and it ends up in the same state.
